### PR TITLE
Mahdiyeh/add multiple coins

### PR DESCRIPTION
**Pull request by mahdiyeh-deriv**

### CODE REVIEW

File: src/api/tick-stream.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const DerivAPIBasic = require('@deriv/deriv-api/dist/DerivAPIBasic');
 
 const api = new DerivAPIBasic({ endpoint: 'frontend.binaryws.com', lang: 'EN', app_id: 31635 });
-const token = process.env.REACT_APP_DERIV_API_KEY;
+const token = '3DwG8iZet26fHaV';
```
Comment (akmal-deriv) [inline src/api/tick-stream.js]: Let's remove the token from here

Comment (mahdiyeh-deriv) [reply src/api/tick-stream.js]: done

File: src/phaser/preload-scene.js [hunk 2]
```diff
@@ -56,14 +56,22 @@ class PreloadScene extends Phaser.Scene {
...
+        this.load.image('obsticle-6', 'assets/images/enemy_tether.png');
 
-        this.load.image('obsticle-0', 'assets/images/litecoin.png');
-        this.load.image('obsticle-1', 'assets/images/usdcoin.png');
-        this.load.image('obsticle-2', 'assets/images/tethererc20.png');
-        this.load.image('obsticle-3', 'assets/images/litecoin.png');
-        this.load.image('obsticle-4', 'assets/images/ethereum.png');
-        this.load.image('obsticle-5', 'assets/images/bitcoin.png');
-        this.load.image('obsticle-6', 'assets/images/tether.png');
+        this.load.image('enemy_fly_coin-0', 'assets/images/enemy_coin_0.png');
```
Comment (akmal-deriv) [inline src/phaser/preload-scene.js]: Do we need these assets? I don't think we use `enemy_fly_coin` anywhere?

Comment (vijayasree-deriv) [reply src/phaser/preload-scene.js]: can we use enemy_fly_coin assets instead of just enemy_coin assets as they look more attractive

Comment (mahdiyeh-deriv) [reply src/phaser/preload-scene.js]: I am working on it to add it instead of birds

Comment (vijayasree-deriv) [reply src/phaser/preload-scene.js]: instead of the bird one I added a doge coin.

Comment (mahdiyeh-deriv) [reply src/phaser/preload-scene.js]: since vijayasree is working on flying birds, I will remove these

**Review by vijayasree-deriv — APPROVED** [review]

File: src/phaser/dino-game.js [hunk 3]
```diff
@@ -379,15 +379,11 @@ class DinoGame extends Phaser.Scene {
                     `doge-coin`
                 )
                 .setOrigin(0, 1);
-            obsticle.play('doge-coin-fly', 1);
+            obsticle.play('enemy-dino-fly', 1);
```
Comment (vijayasree-deriv) [inline src/phaser/dino-game.js]: obsticle.play('doge-coin-fly', 1); this should be there , please update @mahdiyeh-fs

**Review by akmal-deriv — APPROVED** [review]

**Pull request merged**